doc/classes/KinematicBody2D.xml

```diff
@@ -17,6 +17,7 @@
 			<return type="Vector2">
 			</return>
 			<description>
+				Returns the velocity of the floor. If the floor is not a KinematicBody2D, this will always be [code]Vector2(0,0)[/code]. This value is only updated when [method move_and_slide] is called.
```

Comment:
Avoid the first passive:
`This value is only updated when [method move_and_slide] is called.` -> `This value only updates when [method move_and_slide] is called.`